src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopExeWtihNetStandardLib.cs

```diff
@@ -328,14 +310,32 @@ public void It_does_not_include_netstandard_when_inbox(bool isSdk)
                 .Pass();
 
             var outputDirectory = isSdk ?
-                buildCommand.GetOutputDirectory("net461") :
+                buildCommand.GetOutputDirectory("net471") :
                 buildCommand.GetNonSDKOutputDirectory();
 
             outputDirectory.Should().OnlyHaveFiles(new[] {
                 "TestApp.exe",
                 "TestApp.pdb",
                 "TestLibrary.dll",
-                "TestLibrary.pdb"
+                "TestLibrary.pdb",
+
+                //  These DLLs are still included in the output folder when targeting .NET 4.7.1
+                //  With .NET 4.7.2, they are not included anymore
```

Comment:
@joperezr @AlexGhiondea Is it correct to expect all of these files in the output directory for a .NET 4.7.1 project which depends on a .NET Standard 2.0 library?

Reply:
Yes. As long as you target 471 and consume a .Net Standard assembly (2.0 or 1.x) you will need these 12 extra shims in your output folder. If you instead target 4.7.2, you shouldn't need them any longer.

Reply:
Turns out I think we can delete this test entirely.  The scenario is covered by `GivenThatWeWantToTargetNet471.It_builds_a_net471_app_referencing_netstandard20`